account_invoice_pricelist_sale/README.rst

```diff
@@ -71,7 +71,7 @@ Contributors
 Other credits
 -------------
 
-The migration of this module from 15.0 to 16.0 was financially supported
+The migration of this module from 17.0 to 18.0 was financially supported
 by Camptocamp
```

Comment:
Please don't edit those strings. Only migration to 16.0 was `financially supported`. v18.0 migration is made by camptocamp.